src/github.ts

```diff
@@ -126,15 +125,11 @@ async function downloadFromGithub(
     release = undefined
   }
 
-  if (release !== undefined) {
-    addAddonName(addon, release.name)
```

Comment:
`addAddonName` is now called further up after creating the releases, no need for the specific `updateX` functions to know that we track names.